tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_cell_data.py

```diff
@@ -26,16 +26,10 @@ def test_enable_cell_data_via_settings_app(self):
 
         # deal with prompt that sometimes appears (on first setting)
         if cell_data_prompt.is_displayed:
+            self.assertTrue(cell_and_data_settings.is_data_toggle_checked)
```

Comment:
Why do we check the opposite now? The comment below says it shouldn't turned on before we confirm